docs/index.rst

```diff
@@ -4,4 +4,315 @@ JDBC Connector
 ==============
 
 The JDBC connector allows you to import data from any relational database with a
-JDBC driver into Kafka topics.
+JDBC driver into Kafka topics. By using JDBC, this connector can support a wide variety of
+databases without requiring custom code for each one.
+
+Data is loaded by periodically executing a SQL query and creating an output record for each row
+in the result set. By default, all tables in a database are copied, each to its own output topic.
+The database is monitored for new or deleted tables and adapts automatically. When copying data
+from a table, the connector can load only new or modified rows by specifying which columns should
+be used to detect new or modified data.
+
+Quickstart
+----------
+
+To see the basic functionality of the connector, we'll copy a single table from a local SQLite
+database. In this simple example, we'll assume each entry in the table is assigned a unique ID
+and is not modified after creation. Start by creating a database (you'll need to install SQLite
+if you haven't already)::
+
+   $ sqlite3 test.db
+   SQLite version 3.8.10.2 2015-05-20 18:17:19
+   Enter ".help" for usage hints.
+   sqlite>
+
+Next in the SQLite command prompt, create a table and seed it with some data::
+
+   sqlite> CREATE TABLE accounts(id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL, name VARCHAR(255) NOT NULL);
+   sqlite> INSERT INTO accounts(name) VALUES('alice');
+   sqlite> INSERT INTO accounts(name) VALUES('bob');
+
+Now, create a configuration file that will load data from this database in
+``quickstart-sqlite.properties``::
+
+   name=test-sqlite-jdbc-autoincrement
+   connector.class=io.confluent.connect.jdbc.JdbcSourceConnector
+   tasks.max=1
+   connection.url=jdbc:sqlite:test.db
+   mode=increasing
+   increasing.column.name=id
+   topic.prefix=test-sqlite-jdbc-
+
+The first few settings are common settings you'll specify for all connectors. The ``connection.url``
+specifies the database to connect to, in this case a local SQLite database file. The ``mode``
+indicates how we want to query the data. In this case, we have an auto-incrementing unique
+ID, so we select ``increasing`` mode where each query uses the largest ID previously seen to limit
+new results to newly created rows. The ``increasing.column.name`` setting specifies the column in
```

Comment:
I don't think it was *that* long since I also wanted to specifically mention each JDBC connector config in the file. However, I've cleaned this up and renamed to incrementing since I agree that's probably clearer. We'll need a follow up patch in muckrake as well.